examples/aws-test-cost-explorer.yml

```diff
@@ -36,7 +45,14 @@ push-to-kinesis:
       eventType: "public-cloud-records"
       producedBy: "datapull.v1"
       timestamp: ${pipeline.timestamp}
-    quantity: 1
-    period: ${pipeline.time.year}${pipeline.time.month}
-    inUseBy: ${rawData.Certificate.InUseBy}
-    status: ${rawData.Certificate.Status}
+      batchId: ${pipeline.uuid}
+      messageId: ${uuidv4()}
+
+    billingPeriod: ${moment(rawData.TimePeriod.Start).format('YYYYMM')}
+    start: ${rawData.TimePeriod.Start}
+    end: ${rawData.TimePeriod.End}
+    estimated: ${Boolean(rawData.Estimated)}
```

Comment:
Why are we using `Boolean` here? If the `rawData.Estimated` is boolean then we can pass it as-is and otherwise we should probably have a check instead of coercion.

Reply:
@mishok13 i've simplified the semantics of this transformer and renamed it. added comments as well

Reply:
Removed `Boolean()` wrapper here. I was trying to solve a problem of having these values as strings in a final message, but this has to be check in the pipeline runner and not in the yaml file